app/src/main/java/dev/shreyaspatil/foodium/ui/details/PostDetailsActivity.kt

```diff
@@ -37,6 +41,8 @@ import kotlinx.coroutines.ExperimentalCoroutinesApi
 @ExperimentalCoroutinesApi
 class PostDetailsActivity : BaseActivity<PostDetailsViewModel, ActivityPostDetailsBinding>() {
 
+    private var getPost: Post? = null
```

Comment:
Instead of name `getPost` please rename it to `post` Also, make it `private lateinit var`.